src/domain/projects.rs

```diff
@@ -13,7 +13,6 @@ pub struct Project {
 #[derive(sqlx::FromRow, Debug, Clone, PartialEq, Deserialize)]
 pub struct ProjectData {
     pub main_document_id: i32,
-    pub owner: i32,
```

Comment:
Why don't we have the owner field?

Reply:
ProjectData struct keeps the structure of  `post projects` body, in which the value of the owner doesn't need to be stored, as we pass it by the X-User-Id header ( I assume that the project creation can only be done by a user, that automatically claims to be its owner)